### PR TITLE
Update Integrations Page with Current Integrations

### PR DESCRIPTION
**Pull request by jeremyw24**
I have revised the integrations page based on currently AVAILABLE integrations.

### CODE REVIEW

**Review by kimburgess — CHANGES_REQUESTED** [review]
Couple of org / product names to add into `.textlintrc` before merge.

**Comment by mghill** [comment]
Updated textlintrc to include 2 more proper nounds and changed SVSi -> SVSI (couldn't find any instances online of the former?)

File: reference/supported-integrations.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Or, you can refer to the [How to Write A Driver](../tutorial/write-a-driver) gui
 
 ## Messaging
 * Email (MS Graph, Gmail, SMTP, AWS SES)
-* SMS (MessageMedia)
+* SMS (MessageMedia, Whispir, Amazon SMS)
```
Comment (kimburgess) [inline reference/supported-integrations.md]: Where did the Amazon reference come from? I assume this is potentially reference Amazon _SNS_ (which can be used to send SMS's), but cannot see an active integration for this.

Comment (jeremyw24) [reply reference/supported-integrations.md]: Correcto, its been done for NSWH just needs to be pulled out of private, but it exists so happy for it to be on the list.

**Comment by mghill** [comment]
Changed SMS to SNS in that one line. That makes it all ready to merge IMO

**Pull request merged**